app/controllers/words_controller.rb

```diff
@@ -6,8 +6,16 @@ class WordsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @search = Word.ransack(params[:q])
-    @search_words = @search.result(distinct: true).page(params[:page]).per(3) unless params[:q].nil?
+    @projects = Project.all
+    return if params[:q].nil?
+
+    if params[:search_project].nil?
```

Comment:
Dùng ransack để search được được không?
Nếu dc sẽ không cần if else thế này nữa

Reply:
Dung nhu nao the anh?